utils/shared.js

```diff
@@ -1540,10 +1540,27 @@ const filterSelectedFields = (dataObjArray, selectedFieldsArray) => {
     });
 }
 
-const getTemplateForEmailLink = (email, continueUrl) => {
-    return `
+const getTemplateForEmailLink = (
+    email,
+    continueUrl,
+    preferredLanguage = fieldMapping.english.toString()
+) => {
+    return preferredLanguage !== fieldMapping.english.toString()
```

Comment:
Bug can happen here: all values not equal to "English" will return Spanish email template, which is not ideal.

Reply:
Solved